docs/modules/SDDs/pages/0023-managed-services-controller.adoc

```diff
@@ -0,0 +1,89 @@
+= SDD 0023 - Managed Services Controller
+
+:sdd_author:    Tobias Brunner
+:sdd_owner:     
+:sdd_reviewers: 
+:sdd_date:      2020-06-22
+:sdd_status:    draft
+include::partial$meta-info-table.adoc[]
+
+[NOTE]
+.Summary
+====
+To automatically manage application services indepent of where the service instance definition is coming from a managed services controller is introduced.
+====
+
+== Motivation
+
+For automatically generating supporting Kubernetes objects to objects which define application service instances - as defined in xref:0022-managed-services-in-cluster.adoc[0022 - Managed Services in Cluster] - a new controller is introduced. This controller watches for known Kubernetes objects and generates supporting Kubernetes objects like backup schedules or monitoring configuration.
+
+=== Goals
+
+* Controller to enable managed services on a Kubernetes cluster
+
+=== Non-Goals
+
+* Define the content of the supporting objects
+
+== Design Proposal
+
+A Kubernetes controller watches for a defined set of Kubernetes objects and acts when they're created, updated or deleted. The controller acts as a templating engine, meaning it can generate a list of Kubernetes object and fill in needed information in placeholders. The templates and the definition which objects to watch is defined as custom resources which configures the controller.
```

Comment:
```suggestion
A Kubernetes controller watches for a defined set of Kubernetes objects and acts when they're created, updated or deleted. The controller acts as a templating engine, meaning it can generate a list of Kubernetes object and fill in needed information in placeholders. The templates and the definition which objects to watch are defined as custom resources which configure the controller.
```